doc/nrf/app_memory.rst

```diff
@@ -62,19 +68,28 @@ Complete the following actions to optimize the Bluetooth® part of your applicat
 
 For reference, you can find minimal footprint configurations of the :ref:`peripheral_lbs` sample in :file:`nrf/samples/bluetooth/peripheral_lbs/minimal.conf` and the :ref:`peripheral_uart` sample in :file:`nrf/samples/bluetooth/peripheral_uart/minimal.conf`.
 
+.. _app_memory_matter:
 
-Thread
+Matter
 ******
 
-Complete the following actions to optimize the Thread part of your application:
+Besides applying `General recommendations`_, you can also complete the following actions to optimize the :ref:`Matter <ug_matter>` part of your application:
 
-* Disable features that your application does not use.
-  For example, disable the following features:
+* Make sure Zephyr's :ref:`zephyr:shell_api` are disabled for your application.
+  These are listed in a dedicated section in each Matter sample's :file:`prj.conf` file.
+* Check `Thread`_ memory footprint optimization actions, as the Matter application layer uses OpenThread.
+
```

Comment:
Some other optimizations that can be made:

- Change default log level of Zephyr modules from info to warning (CONFIG_LOG_DEFAULT_LEVEL=2). That can be done assuming user logs doesn't use default log level, but define own one.
-  Change log level of Matter logs from debug to info (CONFIG_MATTER_LOG_LEVEL_INFO=y)
- Change verbosity of assert messages to lower value (CONFIG_ASSERT_VERBOSE=n)